gammapy/irf/energy_dispersion.py

```diff
@@ -717,6 +717,8 @@ def get_response(self, offset, e_true, e_reco=None):
             Reconstructed energy axis
         offset : `~astropy.coordinates.Angle`
             Offset
+        oversampling : int, optiona;
+            Migra oversampling factor for each bin of e_reco, default=10
```

Comment:
typo: "optiona" -> "optional".

Currently you have `oversampling=100` in the function signature, and `default=10` in the docstring.
Suggest to never mention defaults in the docstring for that reason, it's hard to get right, and even harder to maintain.

Also, I would suggest an oversampling default of 10, not 100 here.